src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -894,6 +901,7 @@ fn run(mut args: Args) -> Result<(), anyhow::Error> {
             ),
             secrets_reader_aws_prefix: Some(aws_secrets_controller_prefix(&args.environment_id)),
         },
+        enable_persist_txn_tables: true, // WIP
```

Comment:
note to self: hook to up to the flag before merge (once done iterating on CI with the flag on)

Reply:
Just to confirm, the plan is to update this to `enable_persist_txn_tables: args.enable_persist_txn_tables` before merging? I'm going to hold off on an explicit approval until it's changed just to be extra sure we don't accidentally merge as is.

Reply:
Yup, that's the plan! I've started working on getting CI to fully pass in some followups, so I'll go ahead and push this change now